src/server/config.js

```diff
@@ -20,6 +20,10 @@ const config = {
         index: 'gen3-dev-file',
         type: 'file',
       },
+      {
+        index: 'va_data',
```

Comment:
can we let `command.sh` script add some nested fields to `gen3-dev-subject`? So that other developers could also test it locally.

Reply:
To learn Node, I was coding the generator for Elasticsearch data, maybe we can use it in `uc-cdis` for data generation:

1. Clone https://github.com/m0nhawk/es-data-generator/tree/feat/cli
2. Install dependencies: `npm i`
3. Run like this:
```
./main.js -i etl_mickey_0 -h http://localhost -p 9200 -n 10 -d patients --field_value 'auth_resource_path=/programs/DEV/project/test'
```

`field_value` takes key-value pairs separated with `;` (e.g. `auth_resource_path=/programs/DEV/project/test;gender=female;age=1000`).